test/data/match-path-data.ts

```diff
@@ -107,9 +104,7 @@ export const tests: ReadonlyArray<OneTest> = [
     packageJson: { main: "./kalle.js" },
     requestedModule: "lib/mylib.js",
     extensions: [".ts", ".js"],
-    expectedPath: removeExtension(
-      join("/root", "location", "mylib.js", "kalle.js")
-    ),
+    expectedPath: join("/root", "location", "mylib.js", "kalle.js"),
   },
   {
     name:
```

Comment:
This seems to be a test for preventing what do in this PR. Probably this test was added for a reason but I'm not sure what...